cypress/e2e/Crosscutting/BackToTopTest.feature

```diff
@@ -47,45 +47,47 @@ Feature: As a user, I want to be able to use back to top arrow for easy page nav
             | /espanol/cancer/tratamiento/efectos-secundarios/fatiga/fatiga-pro-pdq       | tablet           |
             | /espanol/noticias/comunicados-de-prensa/2018/leucemia-llc-ibrutinib-estudio | desktop          |
 
-    Scenario Outline: Back to top arrow  does not appear on mobile for the English site.
+    Scenario Outline: Back to top arrow appears as an arrow on mobile
         Given screen breakpoint is set to "mobile"
         When user is navigating to "<url>"
-        When user scrolls 601 px "down"
-        Then back-to-top arrow is not displayed
+        When user scrolls 5 px "down"
+        Then back-to-top arrow is displayed with the following svg "/profiles/custom/cgov_site/themes/custom/ncids_trans/dist/.svg"
```

Comment:
Hrmm... this is kind of useless here. You are basically looking for any svg. So I can have a svg of a goat and this would pass. So, I wonder if it is needed at all...

Otherwise you save a copy of the arrow and compare the two SVGs.

Also this is completely an NCIDS thing, so I think you can not care about the SVG here. Like most of these tests should be in the `ncids-js-testing` site.

Reply:
Then I will take an advice and not gonna care about an svg :) 
removing the step